packages/isomorphic/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@xrplf/isomorphic",
+  "version": "1.0.0",
+  "description": "A collection of isomorphic and tree-shakeable crypto hashes and utils for xrpl.js",
+  "keywords": [
+    "crypto",
+    "isomorphic",
+    "xrpl"
+  ],
+  "scripts": {
+    "build": "tsc --build ./tsconfig.build.json --force",
+    "test": "npm run build && jest --verbose false --silent=false ./test/*.test.ts",
+    "test:browser": "npm run build && karma start ./karma.config.js --single-run",
```

Comment:
Is the `--single-run` needed when it's in the karma config?

Reply:
I just confirmed it is not.  This was copied over from xrpl's `package.json`.  I will remove it in #2466 from the other spots.